### ISSUE TITLE
Add --in-file flag to write command

### ISSUE THREAD

**Issue by mackenbach** [secrethub/secrethub-cli#35]
Not every environment handles piping nicely and we cannot predict how users will use our CLI. For instance, a docker container with the `secrethub` binary as entrypoint cannot accept piped content (without workarounds).

## Proposed docs changes

The `write` docs would change by adding this flag:

```
-i, --input-file (string)

   Use the contents of a file as secret input. 
```

And the order in which the command reads the secret from different inputs would change to:

1. `--input-file` OR `--clip`. These two are mutually exclusive and should error when both are set.
1. If input is piped, read from `stdin`
1. If no other input is given, ask the user to type in the secret

**Comment by mackenbach** [comment]
To be consistent with the inject command, let's call it `--in-file`.

**Comment by SimonBarendse** [comment]
This is released in [v0.30.0](https://github.com/secrethub/secrethub-cli/releases/tag/v0.30.0)
